doc/api/dns.md

```diff
@@ -73,14 +76,59 @@ The difference between `REVERSE_A`, `REVERSE_AAAA` and `PTR` is that
 an IPv6 address, while for `PTR` you need to construct yourself the
```

Comment:
Nit: just a little above here, I'd amend the parenthesis talking about `QueryTypeId::A` such that the end of it reads `QueryTypeId::A, QueryTypeId::AAAA`. I'd do this to suggest that for each query as a string there is a corresponding type that is represented by a class.